### DISCUSSION TITLE
Odoo17 Migration Tool: Bad SQL Query

### DISCUSSION THREAD

**Discussion by aeraterta** [OCA/odoo-community.org#176] [Q&A] [answered]
Hi,

We are currently migrating our odoo15 to odoo17 using on-premise method. Error logs are displayed as provided below when doing the SQL query in the upgrade tool:

![image](https://github.com/user-attachments/assets/5c391dae-681b-4920-b26a-3a69b3289c06)

Note: 

- sale_subscription table is empty
- description column is of text datatype

I would like to ask for thoughts on how do i continue with the upgrade

Cheers, 

Anthony

**Comment by legalsylvain** [comment]
> using on-premise method.

what is "on-premise method" ? it is included in your Odoo Enterprise Edition contract ?

**Reply by aeraterta** [reply]
![image](https://github.com/user-attachments/assets/89d3b9ae-ee20-484f-8074-e88cd30b523d)

On-premise is the odoo term for CLI I think.

Yes, we have the enterprise version of odoo also.

**Comment by legalsylvain** [comment, marked as answer]
I guess you've come to the wrong place then! 

The OCA does not support the EE version of Odoo. Instead, open a ticket on github.com/odoo or contact your sales representative.

Alternatively, you can leave Enterprise Edition, switch to Community Edition, work with an OCA integrator then migrate with OpenUpgrade tool.

Different places, different atmospheres.
